app/src/main/java/com/nervousfish/nervousfish/modules/pairing/DummyQRHandler.java

```diff
@@ -17,13 +17,22 @@
      *
      * @param serviceLocator Can be used to get access to other modules
      */
-    @SuppressWarnings("PMD.UnusedFormalParameter")
     // This servicelocator will be used later on probably
     private DummyQRHandler(final IServiceLocator serviceLocator) {
-        super();
+        super(serviceLocator);
         LOGGER.info("Initialized");
     }
 
+    @Override
+    void write(final byte[] buffer) {
+        //dummy
+    }
+
+    @Override
+    void showWarning() {
```

Comment:
The name showWarning is too vague. What does it mean? That the battery is almost dead, that a contact was only received half, that there is a Man-In-The-Middle attack?